bot/admin/test/xray/src/main/kotlin/XrayClient.kt

```diff
@@ -135,14 +136,17 @@ object XrayClient {
 
     fun getProjectFromIssue(issueKey: String): JiraTestProject {
         val klaxon = Klaxon()
+        val xrayResponse = xray.getIssue(issueKey).execute()
+
+        return if (!xrayResponse.isSuccessful) {
```

Comment:
IMHO it would be more readable to handle successful case first and avoid negation here

Reply:
You're right, I've reversed the condition. Now successful case is handled first.